src/lib.rs

```diff
@@ -1,820 +1,695 @@
-                                           arg2: c_int);
-    pub fn rocksdb_options_set_compaction_style(arg1: *mut rocksdb_options_t,
-                                                arg2: c_int);
-    pub fn rocksdb_options_set_universal_compaction_options(arg1: *mut rocksdb_options_t,
-                                                            arg2: *mut rocksdb_universal_compaction_options_t);
-    pub fn rocksdb_options_set_fifo_compaction_options(opt: *mut rocksdb_options_t,
-                                                       fifo: *mut rocksdb_fifo_compaction_options_t);
-    pub fn rocksdb_compactionfilter_create(state: *mut c_void,
-                                           destructor: ::std::option::Option<extern "C" fn(arg1: *mut c_void)
-                                                                         ->
-                                                                             ()>,
-                                           filter: ::std::option::Option<extern "C" fn(arg1: *mut c_void,
-                                                                                   level: c_int,
-                                                                                   key: *const c_char,
-                                                                                   key_length: size_t,
-                                                                                   existing_value: *const c_char,
-                                                                                   value_length: size_t,
-                                                                                   new_value: *mut *mut c_char,
-                                                                                   new_value_length: *mut size_t,
-                                                                                   value_changed: *mut c_uchar)
-                                                                         ->
-                                                                             c_uchar>,
-                                           name: ::std::option::Option<extern "C" fn(arg1: *mut c_void)
-                                                                         ->
-                                                                             *const c_char>) -> *mut rocksdb_compactionfilter_t;
-    pub fn rocksdb_compactionfilter_destroy(arg1: *mut rocksdb_compactionfilter_t);
+
+    pub fn rocksdb_options_set_max_write_buffer_number(opt: *mut rocksdb_options_t, n: c_int);
+
+    pub fn rocksdb_options_set_min_write_buffer_number_to_merge(opt: *mut rocksdb_options_t, n: c_int);
+
+    pub fn rocksdb_options_set_max_write_buffer_number_to_maintain(opt: *mut rocksdb_options_t, n: c_int);
+
+    pub fn rocksdb_options_set_max_background_compactions(opt: *mut rocksdb_options_t, n: c_int);
+
+    pub fn rocksdb_options_set_max_background_flushes(opt: *mut rocksdb_options_t
```

Comment:
I don't think this should exist, but if you were going to do it this way, I think this needs to be `#[repr(c_uint)]`.

Or are these enums the output of the generator?